editProfile.php

```diff
@@ -1,5 +1,6 @@
 <?php
     // Author: Lauren Knight
+    //comment this for ya.
     // Description: Profile edit page
```

Comment:
Are errors being handled? Does it display a message to the user what specific fields are incorrect?

Reply:
i think we're detecting them and stopping them but i don't think we're leaving messages or comments about them. we should add those ya